openpype/hosts/maya/api/plugin.py

```diff
@@ -84,44 +84,6 @@ def get_reference_node_parents(ref):
     return parents
 
 
-def get_custom_namespace(custom_namespace):
-    """Return unique namespace.
-
-    The input namespace can contain a single group
-    of '#' number tokens to indicate where the namespace's
-    unique index should go. The amount of tokens defines
-    the zero padding of the number, e.g ### turns into 001.
-
-    Warning: Note that a namespace will always be
-        prefixed with a _ if it starts with a digit
-
-    Example:
-        >>> get_custom_namespace("myspace_##_")
-        # myspace_01_
-        >>> get_custom_namespace("##_myspace")
-        # _01_myspace
-        >>> get_custom_namespace("myspace##")
-        # myspace01
-
-    """
-    split = re.split("([#]+)", custom_namespace, 1)
-
-    if len(split) == 3:
-        base, padding, suffix = split
-        padding = "%0{}d".format(len(padding))
-    else:
-        base = split[0]
-        padding = "%02d"  # default padding
-        suffix = ""
-
-    return lib.unique_namespace(
-        base,
-        format=padding,
-        prefix="_" if not base or base[0].isdigit() else "",
-        suffix=suffix
-    )
```

Comment:
Did you forget to reimport this function into `plugin.py`? The `ReferenceLoader` in this file is using this function - with this moved I suspect this will now break.

Reply:
I thought we import every thing from lib.
https://github.com/ynput/OpenPype/blob/develop/openpype/hosts/maya/api/plugin.py#L19
That's one of the reasons I moved the function.

Reply:
We import the `lib` but not the functions directly. You'll need to change the calls to it now by doing `lib.get_custom_namespace`. So you should refactor that lower in this file to that.

Reply:
Oops. Sorry. Fixed that one.